elastic/kibana
# [Stack Monitoring] migrate elasticsearch node overview page to React

https://github.com/elastic/kibana/issues/113339
https://github.com/elastic/kibana/issues/112822

Migrates Elasticsearch -> Nodes -> Overview page

- Breadcrumbs will not be correct as its being done separately
- The tab menu is not correct and I will implement it when migrating the Nodes -> Advanced view .  In this PR it continues to use the Nodes view menu. This is what it will look like after updated:
![Screen Shot 2021-09-28 at 1 43 08 PM](https://user-images.githubusercontent.com/1676003/135149077-edacb9dd-0796-4e9b-a576-e5a0c0838bc6.png)
- Instead of using the existing React components which have angular functionality embedded in them i created new components of the same name appended with "React" which can be removed when "Angular mode" is no longer needed.

### Test
- Access the page by going to Elasticsearch Nodes listing.  Click on one of the nodes (pick one with shards):
<img width="857" alt="Screen Shot 2021-09-28 at 6 36 22 PM" src="https://user-images.githubusercontent.com/1676003/135175051-48ceea51-25d1-4c1d-8f18-ab59cfcb86be.png">

- Make sure the page updates with new data when it auto refreshes (I used Edge to get some data)
- Clicking on "Filter for system indices" under Shard Legend to hide or show system indices should work update correctly
- make sure you can drag and zoom into the charts
